README.md

```diff
@@ -272,7 +272,7 @@ In this section, we'll add the necessary configuration to work with GitHub Actio
     mv README-PROVIDER.md README.md
     ```
 
-1. *For Pulumi engineers only:* The final build step in your Action, `publish_SDKs` will fail the first time. 
+1. The final build step in your Action, "Publish SDKs", may fail the first time.
```

Comment:
Suggestion: Add "If publishing the npm package fails, perform the following steps:"